### DISCUSSION TITLE
Help, my remarkable is frozen on startup

### DISCUSSION THREAD

**Discussion by MN-company** [toltec-dev/toltec#442] [Support] [answered]
Hi everyone, after I installed toltec and tried to install (to verify that everything went correctly) the calculator (opkg install calculator), but my remarkable stopped working and crashed, then I restarted it, and it is stuck in the screen “remarkable is starting” I still have access with SSH and therefore I have access to all the files and the terminal, but I don't know how to move, is there anyone who can help me please?

Now in my reMarkable there is no more Toltec installed since I have been a whole day following guides and editing files, I think the problem lies in the "xochilt.service" file since the "systemctl restart xochilt" command gives an error, but I repeat that I am not sure and I do not know where to put my hands to solve the problem.

I would hope that there was a way to restore it without sending it to assistance as it would be a long process and I would need it back in a short time, can anyone help me? Thanks in advance.

![reMarkableFreeze](https://user-images.githubusercontent.com/79520277/134776115-4513fdcd-593d-4c96-bfc4-16a454c93576.jpeg)

**Comment by matteodelabre** [comment]
Hi @MN-company, do not worry, as long as you still have SSH access it is very likely you’ll be able to fix this issue without sending your unit back. Can you tell us what OS version you are running?

**Comment by MN-company** [comment]
Not sure which version I have, but I believe I have beta version 2.10.0.324

**Comment by matteodelabre** [comment]
Installing any app from Toltec automatically installs [rm2fb](https://github.com/ddvk/remarkable2-framebuffer) since it is needed for third party apps to display anything on screen. But rm2fb doesn’t yet support version 2.10 beta. To uninstall rm2fb, run `opkg remove --force-depends display rm2fb-client xochitl`. This will allow Xochitl to run again, but you’ll be unable to use third party apps until rm2fb is updated to support 2.10 or unless you roll back to version 2.9.

**Comment by MN-company** [comment]
Oh ok, I should have been more careful reading the steps, but now I have the problem that opkg is not recognized as a command.
So I tried to reinstall Toltec but giving the command: “wget http://toltec-dev.org/bootstrap” the result is: “wget: Bad address toltec-dev.org”

Reading online they say it is due to the internet connection, is that so?

**Comment by matteodelabre** [comment]
Yes, this is because you don’t have an Internet connection. It’s not clear to me to what extent you have uninstalled Toltec or still have some parts of it lying in your system. Do you still have files in `/opt`?

**Comment by matteodelabre** [comment]
@MN-company Moving this to the discussion support forum which is more appropriate than issues.

**Comment by MN-company** [comment]
Thanks for moving it, I'm not very practical, I still have the /opt folder but no files inside
edit: And I also have the "bootstrap" file in the / home / root directory

**Reply by matteodelabre** [reply]
Do you have the `/home/root/.entware` folder?

**Comment by MN-company** [comment]
Yes, but even that has no files inside

**Comment by matteodelabre** [comment]
Sounds like Toltec is mostly uninstalled then. Do you still have `/etc/systemd/system/xochitl.service.d/toltec-wrapper.conf`?

**Comment by MN-company** [comment]
Yes I have that file.

[Service]
ExecStart =
ExecStart = / opt / bin / xochitl --system

**Comment by matteodelabre** [comment, marked as answer]
Try removing that file and rebooting.

**Comment by MN-company** [comment]
Thanks a lot it worked

**Comment by matteodelabre** [comment]
Yes, it is possible to revert to the last installed version by switching the active partition. Run [this script](https://github.com/ddvk/remarkable-update/blob/main/switch.sh) on your device then reboot.
